packages/cli/src/commands/project/index.ts

```diff
@@ -1,68 +1,93 @@
 import Client from '../../util/client';
 import { parseArguments } from '../../util/get-args';
 import getInvalidSubcommand from '../../util/get-invalid-subcommand';
-import getScope from '../../util/get-scope';
 import handleError from '../../util/handle-error';
-import { help } from '../help';
+import { Command, help } from '../help';
 import add from './add';
 import list from './list';
 import rm from './rm';
-import { projectCommand } from './command';
+import {
+  addSubcommand,
+  listSubcommand,
+  projectCommand,
+  removeSubcommand,
+} from './command';
 import { getFlagsSpecification } from '../../util/get-flags-specification';
 import { ProjectTelemetryClient } from '../../util/telemetry/commands/project';
 import output from '../../output-manager';
+import { getAliases } from '..';
+import getSubcommand from '../../util/get-subcommand';
 
 const COMMAND_CONFIG = {
-  ls: ['ls', 'list'],
-  add: ['add'],
-  rm: ['rm', 'remove'],
+  list: getAliases(listSubcommand),
```

Comment:
I'm making a note that I am planning to rename `getAliases` in https://github.com/vercel/vercel/pull/12516, since there's an existing utility function with that name. I'll wait to merge that until after this PR lands, and will fix up mine appropriately.